lib/configuration/sthConfiguration.js

```diff
@@ -740,5 +740,29 @@ if (ENV.PROOF_OF_LIFE_INTERVAL && !isNaN(ENV.PROOF_OF_LIFE_INTERVAL)) {
             module.exports.PROOF_OF_LIFE_INTERVAL
     );
 }
+if (ENV.PROOF_OF_PERFORMANCE_INTERVAL && !isNaN(ENV.PROOF_OF_PERFORMANCE_INTERVAL)) {
```

Comment:
The new variable should be described in the list at doc/manuals/running.md document.

Reply:
Added the new variable in the list at doc/manuals/running.md document. Fixed in 77610b5